### DISCUSSION TITLE
How is Firebase used?

### DISCUSSION THREAD

**Discussion by creeper-0910** [bluesky-social/atproto#2886] [Bluesky Client App]
When building social-app, a Firebase key is requested, what is it used for?
Also, if I use this key and publish my social-app with changes, is there a possibility that the free Firebase quota will not be enough?
I understand that I have not studied enough, but I would be happy if you could answer this question.

**Comment by bnewbold** [comment]
I believe firebase is used to deliver push notifications to android users. The entire push notification infrastructure was designed to work with independent clients (eg, forks), but I don't think we have it all documented yet.
